modules/Miss.js

```diff
@@ -40,12 +40,18 @@ class Miss extends React.Component {
     return (
       <LocationSubscriber>
         {(location) => {
-          const { render, component:Component } = this.props
+          const { children, render, component:Component } = this.props
           const { noMatchesInContext } = this.state
           const { serverRouter, match } = this.context
           const noMatchesOnServerContext = serverRouter &&
             serverRouter.missedAtIndex(match.serverRouterIndex)
-          if (noMatchesInContext || noMatchesOnServerContext) {
+          const matched = !(noMatchesInContext || noMatchesOnServerContext)
```

Comment:
The `matched` property of `children` indicates whether it's matched or not, doesn't it? So, when no other matches on context, it should be set to `false`. 

I think the `Match`'s `children` behaves the same way. If it's matched, the `matched` property of its `children` is set to `true`, otherwise, `false`.

Please correct me if I understand something wrong.